src/models/MetaModel.ts

```diff
@@ -0,0 +1,82 @@
+export class MetaModel {
+  constructor(
+    public name: string,
+    public originalInput: any) {
+  }
+}
+
+export class ReferenceModel extends MetaModel {
+  constructor(
+    public name: string,
+    public originalInput: any,
```

Comment:
You need to define modifiers for this same properties:

```suggestion
    name: string,
    originalInput: any,
```

Reply:
Ahh 👌 Changed!